doc/dev/Track2TestFramework.md

```diff
@@ -256,4 +256,48 @@ You can use `if (Mode == RecordingMode.Playback) { ... }` to change behavior for
 
 You can use `using (Recording.DisableRecording()) { ... }` to disable recording in the code block (useful for polling methods)
 
+# Support multi service version testing
 
+To enable multi-version testing add the `ClientTestFixture` attribute listing all the service versions to the test class itself or a base class:
+
+```C#
+[ClientTestFixture(
+    BlobClientOptions.ServiceVersion.V2019_02_02,
+    BlobClientOptions.ServiceVersion.V2019_07_07)]
+public abstract class BlobTestBase : StorageTestBase
+{
+    private readonly BlobClientOptions.ServiceVersion _serviceVersion;
+
+    public BlobTestBase(bool async, BlobClientOptions.ServiceVersion serviceVersion, RecordedTestMode? mode = null)
+        : base(async, mode)
+    {
+        _serviceVersion = serviceVersion;
+    }
+
+    // ...
+}
+```
+
+Add a service version attribute to the test class ctor and use the provided service version to create the `ClientOptions` instance.
```

Comment:
Because we have `ServiceVersionAttribute`s elsewhere, maybe phrase this as:

> Add a `ServiceVersion` parameter to the test class .ctor and use the provided service version to create the `ClientOptions` instance.